workspace.spec

```diff
@@ -837,25 +836,21 @@ module Workspace {
 
 		Represents a contributor to the resource.
 
-		Contributors must have a 'contributor_type', either 'person' or 'organization'.
+		Contributors must have a 'contributor_type', either 'person' or 'organization', and
+		a 'name'.
 
-		If the contributor is an organization, the 'name' field must be populated.
-
-		A person must have either 'given_name' and 'family_name' or a 'name'
-		if the name cannot be represented in the given/family name format. Citations will
-		be generated using the initials from the given name(s) and the family name(s).
+		The 'credit_name' field is used to store the name of a person as it would appear in
+		a citation. If there is no 'credit_name' supplied, the 'name' field would be used
+		in citations.
```

Comment:
New -- use `credit_name` to represent a person's name as it would appear in a citation.

Reply:
> If there is no 'credit_name' supplied, the 'name' field would be used in citations.

Hmm, this seems like it's virtually certain to be wrong. Are you sure you don't want to require both?

Reply:
If it's not clear from the `name` field what the most appropriate way to format it for citation is, making the `credit_name` field mandatory would mean the possibility of the `credit_name` being wrong because who/whatever was creating the entry would need to put something in that field. If the field is optional, the name might appear in an inappropriate format in the citation, but the `credit_name` field could be updated without needing to investigate why it had been entered in an incorrect format in the first place.

Reply:
fair enough